packages/expo-screen-orientation/package.json

```diff
@@ -39,9 +39,11 @@
     "@unimodules/core": "*"
   },
   "devDependencies": {
+    "@types/fs-extra": "^9.0.11",
     "expo-module-scripts": "^2.0.0"
   },
   "dependencies": {
-    "@expo/config-plugins": "^2.0.0"
+    "@expo/config-plugins": "^2.0.0",
+    "fs-extra": "^9.0.1"
```

Comment:
should be devDependencies i think?

Reply:
nah it's used in the plugin

Reply:
i updated those libraries to use fs instead of fs-extra, switching out `readFile` and `writeFile` with `readFileSync` and `writeFileSync`